spec/models/user_spec.rb

```diff
@@ -166,15 +166,17 @@
 
   context 'when user has multiple profiles' do
     let(:user) { create(:user, :signed_up) }
+    let(:profile1) { build(:profile, :active, :verified, first_name: 'Jane', user: user) }
+    let(:profile2) { build(:profile, :verified, first_name: 'Susan', user: user) }
 
     before do
-      create(:profile, :active, :verified, first_name: 'Jane', user: user)
-      create(:profile, :verified, first_name: 'Susan', user: user)
+      profile1.save!
```

Comment:
It looks like this is only needed because we want to make sure these records are created for the test. If that is the case, why not inline  these `let` statements and avoid `let` and `before` in this context altogether? Seems like the use of `let` is adding unnecessary complexity for this one spec